src/components/sortable.jsx

```diff
@@ -255,7 +255,7 @@ var Sortable = React.createClass({
         layout: React.PropTypes.oneOf([HORIZONTAL, VERTICAL]),
         padding: React.PropTypes.bool,
         disabled: React.PropTypes.bool,
-        constraints: React.PropTypes.object,
+        constraints: React.PropTypes.shape({}),
```

Comment:
The reason I did that was that `any` can be literally anything like a number or whatever, whereas `shape({})` does at least says that it's an `object` so it's a _tiny_ bit more specific, but I'm happy to change those to `any`s if you'd prefer?